### DISCUSSION TITLE
backup the /home/pi/centaur folder

### DISCUSSION THREAD

**Discussion by Jasiek-dev** [EdNekebno/DGTCentaurMods#24] [General]
Hello,

it will be possible to get copy of backup /home/pi/centaur ?

**Comment by wormstein** [comment]
Due to copyrights we're not gonna publish DGT owned content on this repo. Sorry.

Anyway, centaur software is compiled with Nuitka so there's little chance anyone can see it's content.

**Reply by Jasiek-dev** [reply]
thanks for advise

**Comment by wormstein** [comment]
Is your board broken? What happened to the original sd card?

**Reply by Jasiek-dev** [reply]
I maked chessboard some time ago I did: https://www.hackster.io/lpalcu/foldable-electronic-chess-board-fbe264 but on hall sensor instead of reed switch. Now i'm just trying make my own chessboard like dgt. Where to ask a new question: does anyone know what kind of sensors are installed in DGT Centaur? here or start a new thread?

**Reply by wormstein** [reply]
I'd say to start a new one maybe someone will have a better answer and your question will be lost in here.

Pieces of DGT have a thin foil of aluminium on the bottom so the sensor must be something that reacts on approaching this kind of material.

**Comment by wormstein** [comment]
Ah forget everything, some other in a fork already publisshed an image of the card in his repo. Just use 7 zip to extract...
https://github.com/dsommerfeld/DGTCentaur-2.0-by-dso/tree/master/downloads/V1PGNexport

**Reply by Jasiek-dev** [reply]
Thanks :)
